### PR TITLE
Add Merkrow to contributors

### PR DESCRIPTION
**Pull request by Merkrow**

### CODE REVIEW

File: README.rdoc [hunk 1]
```diff
@@ -1,5 +1,8 @@
 == README
 
+Contributors:
```
Comment (rakvium) [inline README.rdoc:4]: Please use [markup](https://guides.github.com/features/mastering-markdown/) to make a header.

File: README.rdoc [hunk 2]
```diff
@@ -1,5 +1,8 @@
...
+Merkrow
```
Comment (rakvium) [inline README.rdoc:5]: Please add your e-mail if it's suitable for you.

**Comment by Merkrow** [comment]
dispatcher and driver authorization

**Review by rakvium — CHANGES_REQUESTED** [review]

**Comment by rakvium** [comment]
This pull request doesn't make sense now.

**Pull request closed without merging**